SU-HSDO/suhumsci
# STN-493 :  Add Title Region to Three Column with Image layout

# READY FOR REVIEW

## Summary
Add an additional full width field for title above the image for the existing 3 column with image layout. 

Acceptance Criteria:
* The empty field should not impact the layout of any current sites
* Testing at all breakpoints and to meet our browser criteria
* Test with Backstop.js

## Need Review By (Date)
8/20/20

## Urgency
medium

## Steps to Test
1. Run `npm run test` and verify all of our tests pass.
2. The node detail pages use the Three Column with Image Layout. **Review the following node detail pages and confirm they have not changed.** You can compare them to https://sparkbox-sandbox-stage.stanford.edu/. Note that you may see slight differences on staging. I noticed the image on the event node detail page is a different size locally than on staging. That is ok. It may be different on your local environment depending upon when you last synced with staging. 

Expectations prior to any updates in the Drupal admin: The layout of the `<h1>` text aligns to the left and the image aligns to the right.
 - [x] Person: http://sparkbox-sandbox.suhumsci.loc/people/qa-person-all-fields-filled
- [x] Event: http://sparkbox-sandbox.suhumsci.loc/events/1st-year-economics-graduate-student-seminar-series/qa-events-all-fields-filled
- [x] Publications: http://sparkbox-sandbox.suhumsci.loc/publications/qa-publications-all-fields-filled
3. Update a node detail page to use the new title region in the three column with image layout. Let's use the person node detail page. Go to http://sparkbox-sandbox.suhumsci.loc/people/qa-person-all-fields-filled. 
* In the Drupal Admin toolbar, click on `Manage Display`.
* To edit the layout for the page, click on `Manage Layout`
* In the title region, click `Add Block` 
<img width="1213" alt="title-region" src="https://user-images.githubusercontent.com/12678977/90658899-16ead980-e212-11ea-980a-afa2eebdd518.png">
* A menu to choose block will appear to the right. Under `Content fields` select `Full Name`.
* A modal will appear. Set `Label` to `-Hidden-`, `Formatter` to `Heading`, and `Tag` to `H1`. Then click `Add Block`.

* Remove the block that has `Full Name` from the section to the left of the image. 
<img width="736" alt="remove" src="https://user-images.githubusercontent.com/12678977/90661228-ee181380-e214-11ea-9de9-ce924072ecbe.png">

* Save layout and return to the person node detail page.
* Confirm the title region displays `Full Name` above the text and image content.
